src/ui-kit/experimental/hierarchical/selected-result/selected-result.component.html

```diff
@@ -1,17 +1,18 @@
-<ul *ngIf="model"  [attr.aria-label]="configuration.labelText +' results'" class="usa-unstyled-list resultsList" aria-relevant="additions"
-  role="listbox" aria-live="polite">
+<ul *ngIf="model" [attr.aria-label]="configuration.labelText +' results'" class="usa-unstyled-list resultsList"
+  aria-relevant="additions" role="listbox" aria-live="polite">
   <li role="option" *ngFor="let result of model.getItems(); let i = index">
-    <div class="selected-item">
-      <ng-container *ngIf="itemTemplate" [ngTemplateOutlet]="itemTemplate" [ngTemplateOutletContext]="{$implicit:result}">
+    <div [attr.class]="disabled?'selected-item selected-item-disabled' :'selected-item'">
+      <ng-container *ngIf="itemTemplate" [ngTemplateOutlet]="itemTemplate"
+        [ngTemplateOutletContext]="{$implicit:result}">
       </ng-container>
       <ng-container *ngIf="!itemTemplate">
         <span class="header">{{ result[configuration.primaryTextField] }}</span>
         <ng-container *ngIf="configuration.secondaryTextField &&  result[configuration.secondaryTextField] ">
           <span class="secondary-value">{{ result[configuration.secondaryTextField] }}</span>
         </ng-container>
       </ng-container>
-      <span tabindex="0" role="button" class="close fa fa-close" aria-label="Remove Item" aria-hidden="false" (click)="removeItem(result)"
-        (keyup.enter)="removeItem(result)">
+      <span tabindex="0" role="button" [attr.class]="disabled?'close close-disabled fa fa-close' :'close fa fa-close' "
+        aria-label="Remove Item" aria-hidden="false" (click)="removeItem(result)" (keyup.enter)="removeItem(result)">
```

Comment:
@thomastighe That aria-label will hard code that to always be 'Remove Item" as the label. If there are more than one on a page, its a 508 issue. Just making sure you've thought that through by using that tag.

Reply:
No I have not but I will mark this down and will resolve this with the production reediness story